ci/pipelines/deploy-to-staging.yml

```
@@ -537,11 +551,25 @@ groups:
       - deploy-notifications-to-staging
       - smoke-test-notifications-on-staging
       - push-notifications-to-production-ecr
+  - name: stream-s3-sqs
+    jobs:
+      - push-stream-s3-sqs-to-production-ecr
   - name: update-deploy-to-staging-pipeline
     jobs:
       - update-deploy-to-staging-pipeline
 
 jobs:
+  - name: push-stream-s3-sqs-to-production-ecr
+    plan:        
+      - get: stream-s3-sqs-ecr-registry-staging
+        params:
+          format: oci
+        trigger: true
+      - put: stream-s3-sqs-ecr-registry-prod
+        params:
+          image: stream-s3-sqs-ecr-registry-staging/image.tar
+          additional_tags: stream-s3-sqs-ecr-registry-staging/tag
```

Can you add to skip the download after put on these

```suggestion
      - put: stream-s3-sqs-ecr-registry-prod
        params:
          image: stream-s3-sqs-ecr-registry-staging/image.tar
          additional_tags: stream-s3-sqs-ecr-registry-staging/tag
        get_params:
          skip_download: true
```

(same on the put to staging in the deploy-to-test pipeline and actually didn't rememeber in the earlier PR but also on the put to test-ecr)

---

https://github.com/alphagov/pay-ci/pull/646/commits/6ec66f99d6e4dba5ac98826b8addd54c76b97211